src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -258,12 +324,10 @@ public void apply(final VariantContext vc, final ReadsContext readsContext, fina
             validateGVCFVariant(vc);
         }
 
-        for (final ValidationType t : validationTypes) {
-            try{
-                applyValidationType(vc, reportedRefAllele, observedRefAllele, rsIDs, t);
-            } catch (TribbleException e) {
-                throwOrWarn(new UserException.FailsStrictValidation(drivingVariantFile, t, e.getMessage()));
-            }
+        try{
+            applyValidationType(vc, reportedRefAllele, observedRefAllele, rsIDs);
+        } catch (TribbleException e) {
+            throwOrWarn(new UserException.FailsStrictValidation(drivingVariantFile, currentType, e.getMessage()));
```

Comment:
I'm not sure I like this `currentType` field being used to keep track of the annotation we are on. Its a scary global state on the class. If you would like to keep the `applyValidationType()` method being in control of the relevant logic I would recommend that you push the throwOrWarn method into applyValidationType itself so there is no tool-global field of "what was the last validation i tried to execute."

Reply:
Okay, this just came up in a user discussion. A few of these error types generate messages that don't actually indicate which variant is broken. I think it would be incredibly useful for us if you were to append a line to these error messages here that appends the variant context line to the error messages so its clearer what can be done to fix the validation error. I think this is probably higher priority than some of the other comments on this branch. I would also be happy to pull that change out into another PR but I don't want to step on this branchs toes.